Libplanet.Stun/Stun/TurnClient.cs

```diff
@@ -119,7 +117,14 @@ public async Task ReconnectTurn(int listenPort, CancellationToken cancellationTo
                     ClearSession();
                     _turnTaskCts = new CancellationTokenSource();
 
-                    await InitializeTurnAsync(cancellationToken);
+                    await InitializeTurnAsync(cancellationToken).ContinueWith(async t =>
```

Comment:
Can replace this with try catch? Because it awaits task to be finished.

Reply:
Did you mean like this?

```csharp
try
{
    await InitializeTurnAsync(cancellationToken);
}
catch
{
    _logger.Error("Failed to initialize, retry...");
    await Task.Delay(1000)
}
```

(I think it's the same semantic.)

Reply:
Yes

Reply:
Amended https://github.com/planetarium/libplanet/pull/1424/commits/b4eef66334252d6d37ed02712143ed51161413d2